kombu/resource.py

```diff
@@ -144,15 +144,20 @@ def release(self, resource):
     def collect_resource(self, resource):
         pass
 
-    def force_close_all(self):
+    def force_close_all(self, close_pool=True):
```

Comment:
since it is introducing a new API here I'm thinking of BC/including this in a minor version. not fully sure about it right now